test/quic_new_from_listener.c

```diff
@@ -1004,11 +1004,12 @@ static int client_main(int argc, const char *argv[])
     BIO_ADDR_free(bio_addr);
 
     /*
-     * Send signal to parent on error, so it does not get stuck waiting
-     * for I/O.
+     * Send signal to parent on error, so it does not get stuck waiting for
+     * I/O. SIGSTOP signal can not be ignored and forces parent process to
+     * terminate.
```

Comment:
SIGSTOP will stop the parent. Not terminate it.

Reply:
You could kill it with SIGKILL, not sure why it is necessary and why SIGINT or SIGTERM isn't sufficient.